daemon/kill.go

```diff
@@ -55,7 +27,7 @@ func (daemon *Daemon) ContainerKill(job *engine.Job) error {
 		if err := container.KillSig(int(sig)); err != nil {
 			return fmt.Errorf("Cannot kill container %s: %s", name, err)
 		}
-		// FIXME: Add event for signals
+		container.LogEvent("kill signal:" + strconv.Itoa(int(sig)))
```

Comment:
oh, is there a formal process to get events approved?  I assumed this was an old comment and would be easy to fix.  I'll revert it.